torch/quantization/observer.py

```diff
@@ -697,7 +697,7 @@ class HistogramObserver(_ObserverBase):
     min_val: torch.Tensor
     max_val: torch.Tensor
 
-    def __init__(self, bins=2048, upsample_rate=128, dtype=torch.quint8,
+    def __init__(self, bins: int=2048, upsample_rate=128, dtype=torch.quint8,
```

Comment:
Can we also add dtype annotations for upsample_rate and dtype?

Reply:
added.